test/integration/targets/rel_plugin_loading/subdir/inventory_plugins/notyaml.py

```diff
@@ -0,0 +1,168 @@
+# Copyright (c) 2017 Ansible Project
```

Comment:
wonder if a tiny static stub thing that just returns a single fake host might be safer than duplicating yaml inv, but ?